src/components/NavBar/NavBar.vue

```diff
@@ -46,8 +52,37 @@
 				EventBus.$emit('overlay-closeVisibile');
 			});
 		},
+
+		mounted() {	
+			this.$nextTick(() => {
+			window.addEventListener('resize', this.onResize);
+			})
+		},
+		methods: {
```

Comment:
Same comment as before, for consistency, there seems to be a new line missing in this case
```suggestion
		},
		
		methods: {
```